casper/src/test/scala/coop/rchain/casper/util/comm/CasperPacketHandlerSpec.scala

```diff
@@ -381,6 +381,25 @@ class CasperPacketHandlerSpec extends WordSpec {
         test.unsafeRunSync
         transportLayer.reset()
       }
+
+      "respond to ForkChoiceTipRequest messages" in {
+        val request = ForkChoiceTipRequest()
+        val requestPacket =
+          Packet(transport.ForkChoiceTipRequest.id, request.toByteString)
+
+        val test: Task[Unit] = for {
+          tip   <- MultiParentCasper.forkChoiceTip[Task]
+          _    <- casperPacketHandler.handle(local)(requestPacket)
+          head = transportLayer.requests.head
+          _    = assert(head.peer == local)
+          _ = assert(
```

Comment:
You can move this assert into the yield block

Reply:
I wouldn't say this would be an improvement. There's more than one assertion most of the time - I'd rather keep it consistent between single-assertion and multiple-assertion tests, and wouldn't treat the last assertion differently than the others.